### ISSUE TITLE
Lvis object detection pre-trained model

### ISSUE THREAD

**Issue by startupgurukul** [facebookresearch/detectron2#93]
Hello folks, 
Thanks for you detectron2 and special thanks for providing the lvis dataset. So awesome to see something like google-open-images-v5

Coming to the issue,
Noticed there is a pre-trained segmentation model for lvis dataset.
Do you have plans to provide a pre-trained object detection model of lvis dataset?

Thanks in advance.

**Comment by rbgirshick** [comment]
The model zoo contains models trained on LVIS that produce both box and instance segmentation mask output. Does this satisfy what you are looking for? If not, can you clarify what you mean by "object detection model [for] lvis"?

**Comment by startupgurukul** [comment]
Hello Ross, 
Thank you for the quick reply, can only see segmentation model in the model zoo for lvis dataset, couldn't find object detection(just the box) like the one provided for COCO dataset https://github.com/facebookresearch/detectron2/blob/master/MODEL_ZOO.md#coco-object-detection-baselines

![Screenshot from 2019-10-17 08-40-36](https://user-images.githubusercontent.com/11027129/66974791-f023ba00-f0b9-11e9-912f-e282e467796d.png)

**Comment by ppwwyyxx** [comment]
A model that is able to segment the object, is also able to detect (find boxes of) the object.
You can use the segmentation model for just the box.

**Comment by startupgurukul** [comment]
Yes you are right, just that it's significantly slower that just object detection model, there seems to be a a object detection model for coco  but not of lvis dataset.

**Comment by ppwwyyxx** [comment]
As long as you do not predict the mask in the model (`MODEL.MASK_ON=False`), it will be equally fast as a bounding box model.

**Comment by startupgurukul** [comment]
Ok thank you, 

one last naive question before closing the issue, 

 why the .pkl pre-trained model instead of .pt which can also directly  run  on Android and iOS with pytorch mobile.

**Comment by ppwwyyxx** [comment]
pkl is more general as it can be used in any python environment, without the need of pytorch.

**Comment by startupgurukul** [comment]
Thank you :)
